docs/asciidoc/http-client.adoc

```diff
@@ -367,3 +368,69 @@ include::{examplesdir}/uds/Application.java[lines=18..33]
 ----
 <1> Specifies `DomainSocketAddress` that will be used
 ====
+== Host Name Resolution
+The `TCP` client uses by default `Netty's` domain name lookup mechanism that resolves a domain name asynchronously,
+this is as an alternative of the JVM's built-in blocking resolver.
+
+Below is the listing of the available configurations.
+
+[width="100%",options="header"]
+|=======
+| configuration name | description
+| cacheMaxTimeToLive | The max time to live of the cached DNS resource records (resolution: seconds).
+If the time to live of the DNS resource record returned by the DNS server is greater
+than this max time to live, this resolver will ignore the time to live from
+the DNS server and will use this max time to live.
+Default to Integer.MAX_VALUE.
+| cacheMinTimeToLive | The min time to live of the cached DNS resource records (resolution: seconds).
+If the time to live of the DNS resource record returned by the DNS server is less
+than this min time to live, this resolver will ignore the time to live from
+the DNS server and will use this min time to live.
+Default to 0.
+| cacheNegativeTimeToLive | the time to live of the cache for the failed DNS queries (resolution: seconds). Default to 0.
+| disableOptionalRecord | Disables the automatic inclusion of an optional record that tries to hint the remote DNS server about
+how much data the resolver can read per response. By default, this setting is enabled.
+| disableRecursionDesired | Specifies whether this resolver has to send a DNS query with the recursion desired (RD) flag set.
+By default, this setting is enabled.
+| maxPayloadSize | Sets the capacity of the datagram packet buffer (in bytes). Default to 4096.
+| maxQueriesPerResolve | Sets the maximum allowed number of DNS queries to send when resolving a host name. Default to 16.
+| ndots | Sets the number of dots which must appear in a name before an initial absolute query is made.
+Default to -1 which determines the value from the OS on Unix or uses the value 1 otherwise.
+| queryTimeout | Sets the timeout of each DNS query performed by this resolver (resolution: milliseconds). Default to 5000.
+| resolvedAddressTypes | The list of the protocol families of the address resolved.
+| roundRobinSelection | Enables an
+{nettyjavadoc}/io/netty/resolver/AddressResolverGroup.html[AddressResolverGroup] of
+{nettyjavadoc}/io/netty/resolver/dns/DnsNameResolver.html[DnsNameResolver] that supports random selection
+of destination addresses if multiple are provided by the nameserver.
+See {nettyjavadoc}/io/netty/resolver/dns/RoundRobinDnsAddressResolverGroup.html[RoundRobinDnsAddressResolverGroup].
+Default to {nettyjavadoc}/io/netty/resolver/dns/DnsAddressResolverGroup.html[DnsAddressResolverGroup]
```

Comment:
Change "Default to" to "Default:"

Add backticks around `DnsAddressResolverGroup` (within the brackets, so that the word in the link is in the code font).